macGist/GitHubCredential.swift

```diff
@@ -8,21 +8,48 @@
 
 import Foundation
 
-enum GitHubCredential: String {
-    case clientSecret
-    case clientID
+enum GitHubCredentialManager {
+    static private let keychain = Keychain()
     
-    var value: String {
-        guard let path = Bundle.main.path(forResource: "Credentials", ofType: "plist"),
-            let dict = NSDictionary(contentsOfFile: path) as? [String: AnyObject] else { fatalError("No plist found") }
+    static var clientId: String? {
+        get {
+            do {
+                let clientId = try keychain.readGitHubClientId()
+                return clientId
+            } catch {
+                return nil
+            }
+        }
+        
+        set {
+            do {
+                if let id = newValue {
+                    try keychain.save(gitHubClientId: id)
+                }
+            } catch let error {
+                print("Error trying to save client id:: \(error)")
+            }
+        }
+    }
+    
+    static var clientSecret: String? {
+        get {
+            do {
+                let clientSecret = try keychain.readGitHubClientSecret()
+                return clientSecret
+            } catch {
```

Comment:
take a look in this [comment](https://github.com/Bunn/macGist/pull/11/files/9d1102f807c8fc259e10fc34ea24183f1a88e493#diff-3710e79a4e3339cf0255b43d88314eb8)

Reply:
Suggestion accepted.